sdk/identity/azure-identity-broker/tests/test_broker.py

```diff
@@ -30,10 +31,11 @@ def test_username_password_cred_with_broker():
     assert cred._allow_broker
     assert cred._get_app()._enable_broker
 
-
-def test_username_password_cred_without_broker():
     cred = UsernamePasswordBrokerCredential("client-id", "username", "password")
     assert not cred._allow_broker
```

Comment:
This should be:
```suggestion
    assert cred._allow_broker
```
right?